.bazelrc

```diff
@@ -9,8 +9,8 @@ common:ci --color=yes
 
 build:ci --verbose_failures
 build:ci --sandbox_debug
-build:ci --spawn_strategy=standalone
-build:ci --genrule_strategy=standalone
+#build:ci --spawn_strategy=standalone
+#build:ci --genrule_strategy=standalone
```

Comment:
Why is this change needed w/ this fix?

We should probably remove these two configs anyway, but curious why it's needed here. Also, can you please remove these lines rather than commenting them out?

Reply:
Hard to say why it's necessary. It was some time ago when I made this change, I think as an analogy to where I removed the mktemp command and replaced that with a reproducible output dir. When running in the default spawn strategy, that should not harm. Here, I'm enabling use_default_shell_env, maybe that changes the behavior of mktemp? (guessing)

I'll remove the lines altogether.

Reply:
Before I remove them altogether I'll reproduce the failures, by reverting the fix, and will post the link here.

Reply:
Interestingly it's not failing. So I'll leave the revert. Thanks for the comment!